leapp/configs/actor/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+:py:mod:`leapp.configs.actor` represents the import location for private actor config schema that
+are placed in the actor's configs folder.
+
+
+Example:
+      If your actor has a configs folder with a schemas.py python module, import it
+      from the actor like this::
+
+      from leapp.libraries.actor import schemas
```

Comment:
I believe it should be `leapp.libraries.actor`:
```suggestion
      from leapp.configs.actor import schemas
```

Reply:
Sorry I know this is still a draft just noticed it while checking out the code

Reply:
Thanks, resolved.